uffizzi/docker-compose.uffizzi.yml

```diff
@@ -0,0 +1,30 @@
+# This compose file is for demonstration only, do not use in prod.
+version: "3.9"
+
+x-uffizzi:
+  ingress:
+    service: app
+    port: 3001
+
+services:
+  app:
+    depends_on:
+      - "postgres"
+    build:
+      context: ../.
+      dockerfile: ./uffizzi/Dockerfile
+    ports:
+      - 3001:3001
+    environment:
+      ALL_YES: 1
+      NO_INQUIRY: 0
+      TRUST_PROXY_HEADER: 1
+      DB_URL_DEFAULT: postgres://postgres:p0stgr3s@localhost:5432
```

Comment:
Is it even possible to reuse our dockerfile in project root, and set the endpoint env here? Like:

```
build:
  dockerfile: ../Dockerfile
...
environment:
  ...
  ENDPOINT: $UFFIZZI_URL
```

Reply:
You can refer to `.gitpod.yml` to see how it works for GitPod.

Reply:
I will give it a go, this is something I can help with for sure